model/wire.cc

```diff
@@ -490,12 +532,17 @@ namespace
     if (c.size()==4)
       return segNear(c[0],c[1],c[2],c[3],x,y);
     else {
-	  // fixes for tickets 991/1095
+      // fixes for tickets 991/1095
       vector<pair<float,float>> p=allHandleCoords(c);
+      if (!cairoCoords.empty()) {
+        p.clear();
```

Comment:
Could be more simply expressed as p=cairoCoords

Reply:
Thanks, done :) I'm so wary of containers not being populated correctly, since that is how I encounter most segfaults, that I've been overly zealous here.